service/src/main/java/bio/terra/workspace/app/configuration/external/SpendProfileConfiguration.java

```diff
@@ -62,5 +71,45 @@ public String getBillingAccountId() {
     public void setBillingAccountId(String billingAccountId) {
       this.billingAccountId = billingAccountId;
     }
+
+    public Map<String, String> getLimits() {
+      return limits;
+    }
+
+    public void setLimits(Map<String, String> limits) {
+      this.limits = limits;
+    }
+
+    public CloudPlatform getCloudPlatform() {
```

Comment:
I apologize… this has probably been discussed already, but I am confused as to the duplication here of information that is already in the billing profile. Why do we need to do this instead of getting information directly from the billing profile like we do here:

````
spend =
          new SpendProfile(
              spendProfileId,
              getProfileCloudPlatform(profile),
              profile.getBillingAccountId(),
              profile.getTenantId(),
              profile.getSubscriptionId(),
              profile.getManagedResourceGroupId(),
              Optional.ofNullable(profile.getOrganization())
                  .map(SpendProfileOrganization::new)
                  .orElse(null));

Reply:
Since the list of SpendProfiles is only used by tests, I think one could just mock the return value of BPM in tests (for tests that care about limits, which will just be Azure tests). Aren't we getting the limits from BPM itself?

Reply:
We won't be able to get the limits from BPM since we can't rely on having permission to get that information, hence adding it directly to WSM config.  That said, i'm open to a different or better method of getting the SpendProfile with limits from the config.